packages/file-format-ts/index.ts

```diff
@@ -1,7 +1,3 @@
-import * as FileFormat1 from './v1-types.js'
-import * as FileFormat2 from './v2-types.js'
-import * as FileFormat3 from './v3-types.js'
+import * as FileFormat from './types.js'
 
-export { FileFormat1, FileFormat2, FileFormat3 }
-
-export default FileFormat3
+export default FileFormat
```

Comment:
Not sure about the setup and configuration of this project, 

But it might be that you could write just
```ts
export * from './types'
```

Reply:
I _think_ the `.js` extension is there to explicitly import the transpiled JavaScript.

Just tried the `export * from`, seems like the compiler is not happy with it. But it does make me wonder whether the generate can just produce the full package instead of having this extra shell around it that doesn't really do much.

Reply:
I don't think `export * from` is desirable here, since there are uses of the namespaced imported module as `FileFormat.xxxwhatever` later.